src/api/unified/symbol_manager.hpp

```diff
@@ -37,6 +37,14 @@ const int NUM_ENV_VARS = 2;
                     "for instructions to set up environment for Unified backend.", \
                     AF_ERR_LOAD_LIB)
 
+static int backend_index(af::Backend be) {
+    switch (be) {
+        case AF_BACKEND_CPU: return 0;
+        case AF_BACKEND_CUDA: return 1;
+        case AF_BACKEND_OPENCL: return 2;
+        default: return -1;
```

Comment:
The [SymbolManager](https://github.com/arrayfire/arrayfire/blob/devel/src/api/unified/symbol_manager.cpp#L158) and [setBackend](https://github.com/arrayfire/arrayfire/blob/devel/src/api/unified/symbol_manager.cpp#L203) function sets the activeBackend to one of those backends.